app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -289,6 +287,18 @@ public void onClick(DialogInterface dialog, int which) {
         dialog.show();
     }
 
+    // removes filters and shows all shows
+    private void resetShows() {
+        adapter.update(showFilter.getAllShows());
+        rvFeed.smoothScrollToPosition(0);
+
+        // resets everything
+        showFilter.setFilters();
+        selectedGenres = new boolean[]{false, false, false};
+        selectedNetworks = new boolean[]{false, false};
+        selectedYears = new boolean[]{false, false, false};
```

Comment:
I still think it'd probably be useful to try to see if we can remove having this array of booleans, but I can see how it might be beneficial for easily checking selected state of a checkbox without needing to know the title.

Alternatively, you could do instead:
```
val filterName = checkBoxes[which]
val isChecked = selectedGenres.contains(filterName)
```

^this seems a little bit easier to read and less prone to bugs, especially if you ever reorder the genres or check boxes.  Now, just 1 list needs to stay in order compared to 2

Reply:
Just saw the comment on the other PR -- yeah, having the top level arrays will ultimately be more space efficient than creating them each time.  Unfortunate API but not much that you can do